### ISSUE TITLE
Too strong laser on S1

### ISSUE THREAD

**Issue by Scorotron** [arkypita/LaserGRBL#530]
Hi there. Is there any solution how to set weaker laser than S1? I added custom button M3 S1 for focus but it is too strong and burn the small point into wood or paper. S0.1 or S0,1 does not working, the same like S1. I have 2500 mW. Thanks

**Comment by OldShark** [comment]
Hello. I did the same thing with mine. Added a button to focus the LD (M3 S1). my LD is 2.5 w. But I'm not using his driver.

**Comment by arkypita** [comment]
Arduino can modulate PWM only in a range of 0-255 steps. This is an hardware limit so you can not have less then 1/255 of power.

However it sound very strange. I have a 2W blue laser and value less then S30 does not light up the laser at all.

**Comment by Scorotron** [comment]
Thank so much for an answers. I have similar custom button, but M3 instead of M4. S1. But when I changed M4, it is the same. Focused S1 = hole to the paper or wood. But PWM should be working. I tried burn through more sheets of 80g paper and the more "S" parameter, the more pages burned through. On the pict very unfocused S1.

![img_2413](https://user-images.githubusercontent.com/45948850/50378651-9f793700-0637-11e9-8ea9-b18076172a8d.jpg)
